lib/Net/Async/Blockchain/BTC.pm

```diff
@@ -231,7 +231,8 @@ async sub hashblock {
 
     my @transactions = map { $_->{block} = $block_response->{height}; $_ } $block_response->{tx}->@*;
 
-    await fmap_void { $self->transform_transaction(shift) } foreach => \@transactions;
+    await fmap_void { $self->transform_transaction(shift) } foreach => \@transactions,
+        concurrent                                                  => 3;
 
```

Comment:
in Card description it says `5` for ETH & `2` for others.

Reply:
updated the description. Thanks